app/uk/gov/hmrc/directdebitupdateemailbackend/controllers/package.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.directdebitupdateemailbackend
+
+import play.api.mvc.Result
+
+import scala.concurrent.Future
+
+package object controllers {
+
+  implicit def toFuture(result: Result): Future[Result] = Future.successful(result)
```

Comment:
Is this used anywhere?

Reply:
Although it might be useful to have in future, so ignore me

Reply:
yeah, it's being used in one place in SjController to avoid having to write Future.successul. It was being used more in an earlier iteration of my code but will probably see more use in the future I reckon